_notices/rsn0053.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 53 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "RAPIDS 25.10 consolidates Docker images to single tag per CUDA major version"
+notice_author: RAPIDS Ops
+notice_status: "In Progress"
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v25.10"
+notice_created: 2025-09-24
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2025-09-24
+---
+
+## Overview
+
+Starting with RAPIDS `v25.10`, RAPIDS is consolidating Docker images to use a single rolling tag per CUDA major version instead of maintaining separate tags for each CUDA minor version. This change simplifies image selection and reduces maintenance overhead while maintaining compatibility within CUDA major versions.
+
+## Changes
+
+### New Tagging Strategy
+
+RAPIDS will now publish Docker images using the following naming convention:
+
+```
+<RAPIDS_VERSION>-cuda<CUDA_MAJOR_VERSION>-py<PYTHON_VERSION>
+```
+
+For example: `rapidsai/base:25.10-cuda12-py3.13` instead of `rapidsai/base:25.10-cuda12.9-py3.13`.
```

Comment:
```suggestion
For example: `rapidsai/base:25.10-cuda12-py3.13` instead of `rapidsai/base:25.10-cuda12.9-py3.13`.

Available tags for each release can be found at https://hub.docker.com/r/rapidsai/base/tags
```

Let's include a link to help people find the valid tags they can pull.